deepspeed/runtime/config.py

```diff
@@ -805,8 +805,6 @@ def _initialize_params(self, param_dict):
         self.gradient_clipping = get_gradient_clipping(param_dict)
         self.fp16_enabled = get_fp16_enabled(param_dict)
         self.bfloat16_enabled = get_bfloat16_enabled(param_dict)
-        assert not (self.fp16_enabled and self.bfloat16_enabled), 'bfloat16 and fp16 modes cannot be simultaneously enabled'

```

Comment:
Are you sure this line should be deleted? I can't imagine how one would use `bfloat16` and `fp16` at the same time.

Reply:
these lines were duplicated somehow during the merge/conflict resolution. the check still remains below

Reply:
I see that [now](https://github.com/microsoft/DeepSpeed/blob/bf0dd663cab9080a24c1972a40634aec7606f45e/deepspeed/runtime/config.py#L811). Thank you for the clarification.